cmd/node/main.go

```diff
@@ -1192,6 +1202,28 @@ func createNodesCoordinator(
 		return nil, err
 	}
 
+	thresholdEpochDuration := 0.25
```

Comment:
magic number - add to config.

Reply:
also this threshold shoud not be considered if epoch is short. define a constant for short.

Reply:
added to config. Won't add that constant for short epochs, we can change the threshold in config